tap_salesforce/__init__.py

```diff
@@ -134,13 +133,6 @@ def generate_schema(fields, sf, sobject_name, replication_key):
             unsupported_fields.add(
                 (field_name, sf.get_blacklisted_fields()[field_pair]))
 
-        inclusion = metadata.get(
```

Comment:
I don't think you meant to delete this

Reply:
It was only used by the code inside the `if` statement below, and I hope the `get` of the `metadata` doesn't have side effects, so I think it would be unnecessary to keep it.